src/sentry/api/endpoints/project_trace_item_details.py

```diff
@@ -24,6 +24,17 @@
 from sentry.snuba.referrer import Referrer
 from sentry.utils import snuba_rpc
 
+SUPPORTED_TRACE_TYPES = {
+    "ourlogs": SupportedTraceItemType.LOGS,
+    "spans": SupportedTraceItemType.SPANS,
+}
+
+
+DISALLOW_LIST = {
+    "sentry.organization_id",
+    "sentry.item_type",
+}
```

Comment:
Not for this PR but:

I think it would be better that we fully map all 'known' attributes in `OURLOG_ATTRIBUTE_DEFINITIONS` regardless of whether we choose to expose them or not. In some way we can mark the `ResolvedAttribute` (or ResolvedColumn) as a column that it not meant to be exposed to the frontend (eg. internal etc.)

Then we can simply convert similar to using the `translate_internal_to_public_alias` for any TraceItemAttribute.

Reply:
As discussed, I agree but would prefer to get this change in for now and follow-up with that 👍